.cirrus.yml

```diff
@@ -87,7 +87,7 @@ linux_task_template: &LINUX_TASK_TEMPLATE
 # YAML alias for compute credits.
 #
 compute_credits_template: &CREDITS_TEMPLATE
-  # Only use credits for non-DRAFT pull-requests to SciTools/iris master branch by collaborators
+  # Only use credits for non-DRAFT pull-requests to SciTools/iris main branch by collaborators
```

Comment:
[From what I understand about Cirrus](https://cirrus-ci.org/guide/writing-tasks/), `use_compute_credits` will also be true for merges onto feature branches. To achieve what is stated in the comment, we would also need a comparison between `CIRRUS_BASE_BRANCH` and `CIRRUS_DEFAULT_BRANCH`.